src/qc/calculate_qc_metrics/config.vsh.yaml

```diff
@@ -0,0 +1,88 @@
+functionality:
+  name: calculate_qc_metrics
+  namespace: "qc"
+  description: |
+    Add basic quality control metrics to an .h5mu file.
+
+    The metrics are comparable to what scanpy.pp.calculate_qc_metrics output,
+    although they have slightly different names:
+
+    Var metrics (name in this component -> name in scanpy):
```

Comment:
n_counts and n_genes are not in here, but these (I think) as the same as `total_counts_...` idk why scanpy uses both names

Reply:
Probably the `{expr_type}` values were added at some point, but `n_counts` and `n_genes` was kept for backwards compatibility